api/ImageBitmap.json

```diff
@@ -30,12 +30,10 @@
             "version_added": "37"
           },
           "safari": {
-            "version_added": false,
-            "notes": "See <a href='https://webkit.org/b/182424'>bug 182424</a>."
+            "version_added": "15"
```

Comment:
![image](https://user-images.githubusercontent.com/32498324/134072119-1c00edb2-0f40-488c-92d7-ca4737c7c00d.png)

Reply:
Moved to https://github.com/mdn/browser-compat-data/pull/14055